app/pods/spin/index/template.hbs

```diff
@@ -7,29 +7,18 @@
             <img src="https://minio.codingblocks.com/amoeba/spin-n-win-board.svg" alt="Spin-N-Win">
           </div>
           <div class="spin-red font-xxl mb-4 pb-4 bold t-align-c t-align-lg-l">
-            This festive season, Spin the CB Wheel and win assured prizes!!
-          </div>
-          <div class="mb-4 d-flex d-lg-block justify-content-center">
-            <button class="button-dashed button-pink w-75 p-3 border-round" {{did-insert (action (mut spinsLeftBox))}}>
-              <div class="d-flex justify-content-between align-items-center px-3 flex-wrap">
-                <span class="font-xl bold black mx-auto mr-lg-0 mb-2">{{stats.availableSpins}} Spin{{if (gt stats.availableSpins 1) "s"}} Left</span>
-                <a class="font-sm bold spin-red mx-auto mr-lg-0" href="#scrollTagForWinSpin">Earn More <i class="fas fa-long-arrow-alt-right"></i></a>
-              </div>
-            </button>
-          </div>
-
-          <div class="font-md spin-red mb-5 t-align-c t-align-lg-l">
-            Get assured <span class="bold">500 CB Credits</span> on your first successful spin
+            Thank You for your participation in the Spin N Win Contest and making it a success!
           </div>
+          
+          <div class="my-5 font-md spin-red t-align-c t-align-lg-l">
+            We reached a staggering number of students across India, 60,000+ Spins were made, and Secret Santa gave away prizes worth over 5 Lacs during the contest.
 
-          <div class="d-flex d-lg-block justify-content-center">
-            <button class="button-solid button-purple letter-spaced spin-button" disabled={{isSpinning}} {{action (perform spin)}}>
-              <span class="font-xxl bold p-4" >
-                {{if isSpinning 'SPINNING..' 'TAP TO SPIN'}}
-              </span>
-            </button>
+            <p>
+              Don’t forget to make Secret Santa happy by sharing your gifts on social media platforms! <a href="https://twitter.com/search?q=CBSanta">#CBSanta</a>.
+              <br>
+              You can also subscribe to our social media channels for upcoming contests and updates regarding Coding Blocks.
```

Comment:
update with links mailed 

Social media links for static page

https://cb.lk/lisw - Linkedin

https://cb.lk/fbsw - FB

https://cb.lk/insw -Insta

https://cb.lk/twsw - twitter

https://cb.lk/ytsw - youtube